autodispose/src/main/java/com/uber/autodispose/internal/DoNotMock.java

```diff
@@ -17,7 +33,7 @@
 @Documented
 @Retention(RetentionPolicy.CLASS)
 @Target(ElementType.TYPE)
-@interface DoNotMock {
+public @interface DoNotMock {
```

Comment:
Can we avoid making this public, will show up on other folks classpaths and may be somewhat common.

Reply:
Talked offline, not really avoidable unfortunately unless we wanted to make it a separate compileOnly artifact